src/timing.h

```diff
@@ -4,6 +4,7 @@
 #define JL_TIMING_H
 
 #include "julia.h"
+#include <stdbool.h>
```

Comment:
No need for stdbool, just use int (I don't believe we use stdbool anywhere else anyways).

Reply:
I find `bool` a lot clearer for expressing intent. Plus, it behaves as you'd expect in comparisons

However you're right that this isn't used anywhere else so I'll bow to tradition here.